### PR TITLE
React Typescript Created

### PR DESCRIPTION
**Pull request by davirst**
Creating a react set-up with typescript

### CODE REVIEW

File: src/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    root: true,
+    parser: '@typescript-eslint/parser',
+    plugins: ['@typescript-eslint'],
+    extends: [
+        'eslint:recommended',
+        'plugin:@typescript-eslint/recommended',
+        'prettier',
```
Comment (Dada-Tech) [inline src/.eslintrc.js]: if there is no prettier config, remove the plugin please

File: node_modules/.cache/default-development/1.pack
Comment (Dada-Tech) [inline node_modules/.cache/default-development/1.pack:1]: don't commit anything in node_modules. 

also, add the node modules to a .gitignore file and commit it

File: src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import { Routes, Route } from 'react-router-dom';
+import Login from './pages/Login/Login'
+import {GlobalStyle} from "./theme/theme";
+
+
+const App: React.FC = () => {
+  return (
+      <GlobalStyle>
+        <Routes>
+          {/*<Route path="/" element={Login}/>*/}
```
Comment (Dada-Tech) [inline src/App.tsx]: don't commit dummy/commented out code. remove it if it isn't needed anymore

File: src/theme/theme.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { createGlobalStyle } from 'styled-components';
+// import { secondFrameStyle } from 'styled-components';
+
+const GlobalStyle = createGlobalStyle`
+  body {
+    // font-family: 'Poppins', sans-serif;
```
Comment (Dada-Tech) [inline src/theme/theme.tsx]: remove comments if they are not needed

**Review by Dada-Tech — CHANGES_REQUESTED** [review]
The app was able to work with routing when followed the guide from 

https://reactrouter.com/en/main/start/tutorial . 

The code you have in the codebase does not match what ReactRouter says to do.

File: src/App.tsx (hunk 2 above)
Comment (davirst) [reply src/App.tsx]: Just in case I have to create other routes

**Comment by Dada-Tech** [comment]
The node modules cache is still committed which it should not be but I have to merge in the interest of time

**Pull request merged**